<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectiveColor</class>
 <widget class="QWidget" name="SelectiveColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>455</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Preset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="preset"/>
     </item>
     <item>
      <widget class="QToolButton" name="add_preset">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove_preset">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Color range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="range"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cyan</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="slider_cyan">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spin_cyan">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Magenta</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="slider_mag">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="spin_mag">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Yellow</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="slider_yell">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="spin_yell">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Black</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="slider_black">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="spin_black">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="absolute">
       <property name="text">
        <string>Absol&amp;ute</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="relative">
       <property name="text">
        <string>Re&amp;lative</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>122</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_cyan</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_cyan</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_cyan</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_cyan</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_mag</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_mag</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_mag</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_mag</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_yell</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_yell</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_yell</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_yell</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_black</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_black</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_black</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_black</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
